### DISCUSSION TITLE
[mpvnet] 有个地方想问下怎么改

### DISCUSSION THREAD

**Discussion by xiaozixun** [hooke007/mpv_PlayKit#46] [libmpv]
![批注 2021-11-15 170517](https://user-images.githubusercontent.com/22409452/141753594-ff0abe6f-acd3-4fc5-8b23-8ad18abfc302.png)
请问这里应该怎么去掉呢？我用了[https://github.com/cyl0/mpv-osc-morden-x/](url)这里的osc，原本它应该显示标题的。
![批注 2021-11-15 171041](https://user-images.githubusercontent.com/22409452/141754794-de719bfa-47b1-4007-bbbb-c221e779a6b8.png)

**Comment by hooke007** [comment]
除非懒，脚本作者都会写使用说明

https://github.com/cyl0/mpv-osc-morden-x/#how-to-config

**Reply by xiaozixun** [reply]
`showtitle = true,		-- show title and no hide timeout on pause` 
我看了下它这条确实是开启的，但是显示的却是音量、速度、列表，所以我猜测是mpvnet中的某一项。

**Reply by hooke007** [reply]
showtitle 意思是 显示不显示而已

**Reply by hooke007** [reply]
title = '${media-title}',   -- string compatible with property-expansion
                                -- to be shown as OSC title

**Reply by hooke007** [reply]
按照此处重写一遍进设置文件，mpvnet有预置参数会覆盖原始设计，重写一遍覆盖mpvnet的设置

**Reply by xiaozixun** [reply]
> 按照此处重写一遍进设置文件，mpvnet有预置参数会覆盖原始设计，重写一遍覆盖mpvnet的设置

好的，谢谢。

**Reply by xiaozixun** [reply]
之前我出去了……
回来的时候试了试发现还是如主楼的图所示，我特意去下了mpv-lazy试试了也是一样的：那里没有标题。

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/blob/771d0ba1b34e4d564272a361b50408024d2e8cd2/portable_config/script-opts/osc.conf#L50

lazy本来就不显示标题，不改怎么可能显示

**Reply by xiaozixun** [reply]
> 
> 
> https://github.com/hooke007/MPV_lazy/blob/771d0ba1b34e4d564272a361b50408024d2e8cd2/portable_config/script-opts/osc.conf#L50
> 
> lazy本来就不显示标题，不改怎么可能显示

好吧，我把net里的script-opts和scripts文件夹与mpv.conf、mpvnet.conf和profiles.conf文件直接覆盖了一次lazy，结果lazy有了，net还是不行。
话说双击运行mpvnet.exe出来的界面是这个
![批注 2021-11-15 222156](https://user-images.githubusercontent.com/22409452/141797780-edf34b28-f257-451a-a87b-69c071f9f18d.png)
看来是这个lua不支持net？

**Reply by hooke007** [reply]
。。。乱覆盖lazy这个版本里的预设就没有意义了。修改标题只需要改我说的那一行就行。

netCM的标题问题我会尝试修改一下。
现在在mpv.conf里使用这个参数 `script-opts-append=osc-title=你要的标题`

图标显示问题没办法了，要么net那边改，要么脚本作者改

**Reply by xiaozixun** [reply]
> 
> 
> 。。。乱覆盖lazy这个版本里的预设就没有意义了。修改标题只需要改我说的那一行就行。
> 
> netCM的标题问题我会尝试修改一下。 现在在mpv.conf里使用这个参数 `script-opts-append=osc-title=你要的标题`
> 
> 图标显示问题没办法了，要么net那边改，要么脚本作者改

嗯，好的。
主要是它原先的脚本里就有这行，我以为是我的其它操作导致的没法显示 [https://github.com/cyl0/mpv-osc-morden-x/blob/b4b13994556c5845f109ba920745ac09a5c70bac/mordenx.lua#L39](https://github.com/cyl0/mpv-osc-morden-x/blob/b4b13994556c5845f109ba920745ac09a5c70bac/mordenx.lua#L39)
这样不知道能不能看到……

**Reply by hooke007** [reply]
同一设置的优先级 mpv.conf > 脚本设置文件 > 脚本预设

**Reply by xiaozixun** [reply]
> 
> 
> 同一设置的优先级 mpv.conf > 脚本设置文件 > 脚本预设

我知道了。
